### PR TITLE
docs:  google gcp / workspace instructions

### PR DESCRIPTION
**Pull request by desimone**
## Summary

Updated the google guide to be much more deferential to Google's own documentation as their screens have been changing rather constantly. Tried to be more explicit about the structure of the steps. Overall, there's not at ton we can do since the process is split between GCP based OAuth2 credential and service account creation, manually adding an impersonated user for the SA, then adding permissions from GSuite to that domain wide service account. Despite the updates on Google's side, these settings are just as hard to find as before. 

## Related issues

- Related to #1052 
- Related to https://github.com/pomerium/pomerium/pull/1738

## Checklist

- [x] reference any related issues
- [x] updated docs
- [ ] updated unit tests
- [ ] updated UPGRADING.md
- [x] add appropriate tag (`improvement` / `bug` / etc)
- [x] ready for review

### CODE REVIEW

**Comment by desimone** [comment]
For preview: https://deploy-preview-2272--pomerium-docs.netlify.app/docs/identity-providers/google.html

File: docs/docs/identity-providers/google.md [hunk 1]
```diff
@@ -54,26 +93,34 @@ Then, you'll need to manually open an editor and add an `impersonate_user` field
...
-Next we'll delegate G-suite group membership access to the service account we just created .
+Next, we need to give that service account permissionson the GSuite / Workspace side of the house.
 
-1. Go to your G Suite domain's [Admin console](http://admin.google.com/).
-2. Select **Security** from the list of controls. If you don't see Security listed, select More controls 1\. from the gray bar at the bottom of the page, then select Security from the list of controls.
-3. Select **Advanced settings** from the list of options.
-4. Select **Manage API client** access in the Authentication section.
-5. In the **Client name** field enter the service account's **Client ID**.
-6. In the **One or More API Scopes** field enter the following list of scopes: `https://www.googleapis.com/auth/admin.directory.group.readonly` `https://www.googleapis.com/auth/admin.directory.user.readonly`
+1. From your Google Workspace domain's [Admin console](http://admin.google.com/), go to **Main menu** menu**> Security > API controls**.
```
Comment (calebdoxsey) [inline docs/docs/identity-providers/google.md:115]: formatting looks off in the preview. Maybe a duplicate `menu**`?

**Review by calebdoxsey — CHANGES_REQUESTED** [review]

**Comment by wasaga** [comment]
They have command line equivalents for every UI, maybe we should just use them?

**Comment by desimone** [comment]
> They have command line equivalents for every UI, maybe we should just use them?

I think we continue with this format because:

1. It's consistent with our other guides
2. The barrier for entry is lower
3. I'm not sure all of these settings (especially the bizzar-o gsuite scopes) are available. 
4. I'm unconvinced the API churn is any less than the docs.

File: docs/docs/identity-providers/google.md (hunk 1 above)
Comment (desimone) [reply docs/docs/identity-providers/google.md:115]: Thank you, good catch. I think I fixed it.

Comment (desimone) [reply docs/docs/identity-providers/google.md]: Thank you, good catch. I think I fixed it.

**Review by calebdoxsey — APPROVED** [review]

**Pull request merged**